examples/byoc/aws/main.tf

```diff
@@ -27,11 +27,11 @@ resource "redpanda_cluster" "test" {
   tags = {
     "key" = "value"
   }
-  aws_private_link = {
-    enabled         = true
-    connect_console = true
-    allowed_principals = ["arn:aws:iam::123456789024:root"]
-  }
+  # aws_private_link = {
+  #   enabled         = true
+  #   connect_console = true
+  #   allowed_principals = ["arn:aws:iam::123456789024:root"]
+  # }
```

Comment:
NIT: ✂️

Reply:
It's solely there for an example to end users. We can't have it actually run because it will cause the whole thing to explode